cmd/gateway/manta/gateway-manta.go

```diff
@@ -14,7 +14,7 @@
  * limitations under the License.
  */
 
-package cmd
+package manta
```

Comment:
This built and was functional for me and I also couldn't find where this should be imported if it needed to be.

Reply:
it is added under `cmd/gateway/gateway.go`

Reply:
I see that now.